email.rst

```diff
@@ -39,11 +39,16 @@ environment variable in the ``.env`` file:
     # use this to disable email delivery
     MAILER_URL=null://localhost
 
-    # use this to configure a traditional SMTP server (make sure to URL-encode the
-    # values of the username and password if they contain non-alphanumeric characters
-    # such as '+', '@', ':' and '*', which are reserved in URLs)
+    # use this to configure a traditional SMTP server
     MAILER_URL=smtp://localhost:25?encryption=ssl&auth_mode=login&username=&password=
 
+.. caution::
+
+    If the username, password or host contain any character considered special in a
+    URI (such as ``+``, ``@``, ``$``, ``#``, ``/``, ``:``, ``*``, ``!``), you must
+    encode them. See `RFC 3986`_ for the full list of reserved characters or use the
```

Comment:
please copy the link, too!

```
Warning, treated as error:
/home/travis/build/symfony/symfony-docs/email.rst:47: ERROR: Unknown target name: "rfc 3986".
```

Reply:
No need to copy the link? @javiereguiluz

Reply:
Yes. I added the link while merging. Thanks for reporting this.